tests/core/test_structure.py

```diff
@@ -1688,6 +1688,7 @@ def test_calculate_ase(self):
         assert not hasattr(calculator, "dynamics")
         assert self.cu_structure == struct_copy, "original structure was modified"
 
+    @pytest.mark.skip(reason="chgnet is failing with Numpy 1, see #3992")
```

Comment:
Need a closer look, [`chgnet` tests are failing for NumPy 1 as well](https://github.com/materialsproject/pymatgen/actions/runs/10330613130/job/28599854900?pr=3992), I guess all ints are now `int64` in pymatgen, which might not be the case for `chgnet`'s cython code.

Reply:
@AegisIK can you comment on this? is the fix on the `chgnet` side to ensure all ints in the atom/bond graph construction are `np.int64`?

Reply:
Look like this test is passing for latest version of chgnet now!
```
 (venv312) venvyang@Yang-MacBook pymatgen % venv312/bin/pytest tests/core/test_structure.py::TestStructure::test_calculate_ase
.                                                                                                                                             [100%]
=============================================================== slowest 30 durations ================================================================
0.07s call     tests/core/test_structure.py::TestStructure::test_calculate_ase

(2 durations < 0.005s hidden.  Use -vv to show these durations.)
1 passed in 1.20s

(venv312) venvyang@Yang-MacBook pymatgen % pip freeze| grep chgnet
chgnet==0.4.0
```